ccmlib/cluster.py

```diff
@@ -314,6 +315,15 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False,
 
         common.assert_jdk_valid_for_cassandra_version(self.cassandra_version())
 
+        # check whether all loopback aliases are available before starting any nodes
+        for node in self.nodes.values():
+            for itf in node.network_interfaces.values():
+                if itf is not None:
+                    if not common.check_socket_available(itf, return_on_error=True):
+                        addr, port = itf
+                        print "Inet address %s:%s is not available; a cluster may already be running or you may need to add the loopback alias" % (addr, port)
```

Comment:
You will need to `from six import print_`, and then use the `print_` function here, to maintain compatibility across python 2 and 3.